CMakeLists.txt

```diff
@@ -71,6 +72,12 @@ if(WITH_LZO)
   find_package(LZO)
 endif()
 
+if(WITH_XCB)
+  find_package(XCB)
```

Comment:
I think it would be better to make use of CMake's FindX11 and then use the _xcb variables set by it? See https://cmake.org/cmake/help/latest/module/FindX11.html